examples/resources/opslevel_check_repository_file/resource.tf

```diff
@@ -34,7 +34,7 @@ resource "opslevel_check_repository_file" "example" {
   filter           = data.opslevel_filter.tier1.id
   directory_search = false
   filepaths        = ["/src", "/tests"]
-  file_contents_predicate {
+  file_contents_predicate = {
```

Comment:
This will introduce a mandatory change for any customers on the current version, we should use this: https://developer.hashicorp.com/terraform/plugin/framework/handling-data/blocks/single-nested

Reply:
@davidbloss and I talked about this and because of this error message terraform generates its an OK thing to bring forward.

```
│ Error: Unsupported block type
│ 
│   on main.tf line 23, in resource "opslevel_infrastructure" "big":
│   23:   provider_data {
│ 
│ Blocks of type "provider_data" are not expected here. Did you mean to define argument "provider_data"? If so, use the equals sign to assign it a value.
```